api/RTCRtpTransceiverInit.json

```diff
@@ -127,10 +127,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This (`sendEncodings`) was added in https://github.com/WebKit/WebKit/commit/596c6ac5e98a67dd698d5e5f3f93622e1af5e19c which suggests it should be supported and this shouldn't be false.